2_adventures/001/amaliahern/04/fizz_buzz.rb

```diff
@@ -0,0 +1,6 @@
+def fizzbuzz(number)
+  return "fizzbuzz" if(number % 3 == 0 and number % 5 == 0)
```

Comment:
Something to think about... if you chose not to return (yet) you can build your result and return when you are done.  This way you only have to check for each/all states one time.

Consider the case where you end up playing this game with 3 primes or 4 or 5?  How long will this one line get?